src/platform_linux/platform.h

```diff
@@ -328,23 +328,26 @@ extern platform_heap_id     Heap_id;
  *	n   - Number of bytes of memory to allocate.
  * -----------------------------------------------------------------------------
  */
-#define TYPED_MANUAL_MALLOC(hid, v, n)                                         \
+#define TYPED_MANUAL_MALLOC(hid, v, n, mf)                                     \
    ({                                                                          \
       debug_assert((n) >= sizeof(*(v)));                                       \
       (typeof(v))platform_aligned_malloc(hid,                                  \
                                          PLATFORM_CACHELINE_SIZE,              \
                                          (n),                                  \
+                                         (mf),                                 \
```

Comment:
Allocation now receives and returns a `platform_memfrag{} *`, So, macro's call changes.